person.rb

```diff
@@ -13,6 +15,11 @@ def can_use_services?
     of_age? || @parent_permission
   end
 
+  def validate_name
+    corrector = Corrector.new
```

Comment:
- [ ] We should have this instance of Corrector set in the Person class on initialization, you can have an example of how to do so in [here ](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/composition_by_examples.md#composable-classes)this will help you access `Corrector` instance whenever you have an instance of `Person` hence will help you have cleaner codes